discord/interactions/connectCommand.ts

```diff
@@ -43,18 +55,29 @@ export const handleConnectCommand = async (
       });
     } else {
       await interaction.reply({
-        content: `Go to this link : ${config.BASE_URL}/verify/${guildId}/${userId}/${alreadyDiscordMember.customLink} and verify your Starknet identity!`,
+        content: `Go to this link : 
+          ${config.BASE_URL}/verify/${guildId}/${userId}/${alreadyDiscordMember.customLink} and verify your Starknet identity!`,
         ephemeral: true,
       });
     }
   } else {
-    const newDiscordMember = new DiscordMember();
-    newDiscordMember.discordServer = alreadyDiscordServer;
+    var newDiscordMember = new DiscordMember();
+
+    newDiscordMember.DiscordServerId = guildId;
     newDiscordMember.discordMemberId = userId;
     newDiscordMember.customLink = nanoid();
-    await DiscordMemberRepository.save(newDiscordMember);
+
+    for (var _i = 0; _i < allDiscordServerConfigs.length; _i++) {
+      var currentRow = allDiscordServerConfigs[_i];
+      newDiscordMember.DiscordServerConfig = currentRow;
+      newDiscordMember.DiscordServerConfigId = currentRow.id;
+      console.log("le config id est", currentRow.id);
+      await DiscordMemberRepository.insert(newDiscordMember);
+    }
```

Comment:
same, we should probably not duplicate the member for each config but have one member per server and multiple configs per server